itou/templates/approvals/includes/box.html

```diff
@@ -75,7 +77,21 @@
                 {% endif %}
             {% endwith %}
         {% endif %}
-        {% if link_from_current_url|default:None %}
+
+        {% if job_seeker_help_text|default:"" == "waiting_period_with_valid_diagnosis" %}
+            <hr class="my-3 my-md-4">
+            <p class="fs-sm mb-0">
+                Un prescripteur habilité a réalisé un diagnostic d'éligibilité. <strong>Vous pouvez commencer un nouveau parcours.</strong>
+            </p>
+        {% elif job_seeker_help_text|default:"" == "waiting_period_without_diagnosis" %}
+            <hr class="my-3 my-md-4">
+            <p class="fs-sm mb-0">Vous avez terminé un parcours il y a moins de deux ans.</p>
+            <p class="fs-sm my-0">
+                Pour prétendre à nouveau à un parcours en structure d'insertion par l'activité économique vous devez rencontrer un prescripteur
+                habilité : France Travail, Mission Locale, Cap emploi, etc.
+            </p>
+        {% endif %}
```

Comment:
J'hésite avec:
```suggestion
        {% if extra_text|default:"" %}
            <hr class="my-3 my-md-4">
            {{ extra_text }}
        {% endif %}
```
et laisser la vue injecter son `mark_safe`/`format_html` en variable.